ci/tests/general/general_vdisk.py

```diff
@@ -222,3 +229,27 @@ def get_filesystem_location(vpool, vdisk_name):
         else:
             raise RuntimeError('Invalid hypervisor type specified: {0}'.format(hv_type))
         return location
+
+    @staticmethod
+    def get_config_params(vdisk_guid):
+        """
+        :param vdisk_guid: Guid of vdisk to retrieve config params from
+        :return: {} containing config params
+        """
+        task_id = GeneralVDisk.api.execute_get_action('vdisks', vdisk_guid, 'get_config_params')
```

Comment:
Since you want to wait for the result, why not just add wait=True to execute_get_action
Same for set_config_params